app/src/main/res/values/strings.xml

```diff
@@ -254,6 +254,7 @@
   <string name="skip">Skip</string>
   <string name="next_arrow">Next</string>
   <string name="get_started">Get Started</string>
+  <string name="onboarding_slide_dots_content_description">Slide %d of 4</string>
```

Comment:
Don't use `4` directly here. Set this too from viewModel using `totalNumberOfSlides`

Reply:
Done.